Sources/Core/Internal/ClickPlugin.swift

```diff
@@ -5,16 +5,21 @@
  */
 
 import Foundation
-import UIKit
+
+#if os(iOS)
+    import UIKit
```

Comment:
We shouldn't need these since `ClickGestureRecognizer` is typealiased per platform.